cli/cmd/clean.go

```diff
@@ -120,15 +118,22 @@ func internalCleanModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 		status := running.Status(&run)
 		if status.Code == process_utils.ProcessStoppedCode {
 			var statusMsg string
+			var postfix string
+
+			if cmdCtx.Cli.Verbose {
+				postfix = "\n"
```

Comment:
Why is postfix required? Does not log.info print a newline?

Reply:
It prints a newline, but in verbose mode it would make indent from previous record to indicate the start of cleaning the next instance.